backend/test/unit/defence.test.ts

```diff
@@ -471,3 +447,116 @@ test('GIVEN user has configured two defence WHEN resetting one defence config TH
 	expect(matchingCharacterLimitDefence).toBeTruthy();
 	expect(matchingCharacterLimitDefence?.config[0].value).toBe('10');
 });
+
+test(
```

Comment:
the following tests replace [these tests](https://github.com/ScottLogic/prompt-injection/pull/743#discussion_r1448487399) that were removed in Heather's branch